.github/workflows/release-manual.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Generate GitHub Release (manual trigger)
+concurrency:
+  group: release-${{ github.head_ref }}
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        required: true
+        description: Release to generate
```

Comment:
This should be described as an existing release tag and should be good maybe to add an example